apps/web/src/main.tsx

```diff
@@ -3,9 +3,9 @@ import ReactDOM from "react-dom";
 
 import ContextContainer from "@common/components/context/ContextContainer";
 import Toast from "@common/components/Toast/Toast";
-import * as messages from "./lang/frCompiled.json";
 
-import Router from "./components/Router";
+import * as messages from "~/lang/frCompiled.json";
```

Comment:
My VS Code linting doesn't seem to like the absolute references.
![image](https://user-images.githubusercontent.com/8978655/208937045-7d013cbc-6f73-44d0-aa81-23472bb553d2.png)

Reply:
Did you restart the typesctipt and eslint servers in VS Code? That is required for it to pick up the rules for this.